pkg/master/master.go

```diff
@@ -570,7 +620,7 @@ func (m *Master) thirdpartyapi(group, kind, version string) *apiserver.APIGroupV
 }
 
 // getExperimentalResources returns the resources for extenstions api
-func (m *Master) getExtensionResources(c *Config) map[string]rest.Storage {
+func (m *Master) getExtensionResources(c *Config, autoscalerStorage, autoscalerStatusStorage rest.Storage) map[string]rest.Storage {
```

Comment:
How about delegate this to the person who will add next parameters? The change is actually temporary - I expect to remove those two parameters for 1.3 when we will stop supporting for autoscaling objects in extensions/v1beta1.